examples/example5/example5.cpp

```diff
@@ -0,0 +1,162 @@
+// SPDX-FileCopyrightText: Loren Burkholder
```

Comment:
Ditto copyright

Reply:
example1.cpp copy&paste - renamed

Reply:
OK, then I'm fine with leaving the copyright as-is